app/assets/javascripts/app_groups.js

```diff
@@ -39,6 +39,14 @@ var app_groups = {
       $form.submit();
     });
 
+    $("[id^='toggle_app_group_status_']").change(function() {
+      var appGroupId = $(this).data("id");
+      var isChecked = $(this).prop("checked");
+      var $form = $("#form_toggle_app_group_status_" + appGroupId);
+      $form.find("#toggle_app_group_status").val(isChecked);
+      $form.submit();
+    });
+
```

Comment:
does this not exist in the existing implementation? if it doesn't exist, then how we implement the toggle status?